CMakeLists.txt

```diff
@@ -166,6 +169,11 @@ if(NOT ENABLE_GENERIC)
 			set(_M_X86 1)
 			set(_M_X86_64 1)
 			add_definitions(-D_M_X86=1 -D_M_X86_64=1 -msse2)
+			if(ENABLE_QT2)
+				set(ENABLE_GENERIC 1) #Qt requires pie, pie breaks JIT
```

Comment:
This will actually fail since you can't enable generic at this point, it's too late in the build process